### DISCUSSION TITLE
About <Page> tag

### DISCUSSION THREAD

**Discussion by Jafee** [zeit-ui/react#480] [Q&A] [answered]
How to edit `Page` tag's style? such as: padding .

**Comment by unix** [comment, marked as answer]
I have provided [an example](https://codesandbox.io/s/geist-ui-override-styles-25w5m) with three common scenarios. (It's no different than if you look at the elements and add class names)
If you're using the `css in js` solution, it's a little easier.

**Reply by Jafee** [reply]
Thank you very much for your reply and patiently wrote the demo.

The `Page` tag finally generates the `main` tag of the `section` tag. I want to change the padding of the `main` tag. Your demo reminded me, I added classname, so I changed it `#__next .pageClassName main { padding: 0 }`

The UI library is great. Thank you for your contribution.
